README.md

```diff
@@ -5,6 +5,10 @@ It is a part of [OpenVINO™ Training Extensions](https://github.com/opencv/open
 
 The project is based on [Kaiyang Zhou's Torchreid](https://github.com/KaiyangZhou/deep-person-reid) project.
 
```

Comment:
I propose to delete README for paper branch and keep only guide how to run training